frontend/src/scenes/actions/ActionEdit.tsx

```diff
@@ -83,23 +84,40 @@ export function ActionEdit({ action: loadedAction, id, onSave, temporaryToken }:
                     />
                 }
                 caption={
-                    <EditableField
-                        multiline
-                        name="description"
-                        value={action.description || ''}
-                        placeholder="Description (optional)"
-                        onSave={(description) => {
-                            setAction({ ...action, description })
-                            setEdited(!!description)
-                        }}
-                        mode={!id ? 'edit' : undefined /* When creating a new action, maintain edit mode */}
-                        autoFocus={!!id}
-                        data-attr="action-description"
-                        compactButtons
-                        maxLength={600} // No limit on backend model, but enforce shortish description
-                        paywall={!hasAvailableFeature(AvailableFeature.INGESTION_TAXONOMY)}
-                        saveButtonText="Set"
-                    />
+                    <>
+                        <EditableField
+                            multiline
+                            name="description"
+                            value={action.description || ''}
+                            placeholder="Description (optional)"
+                            onSave={(description) => {
+                                setAction({ ...action, description })
+                                setEdited(!!description)
+                            }}
+                            mode={!id ? 'edit' : undefined /* When creating a new action, maintain edit mode */}
+                            autoFocus={!!id}
+                            data-attr="action-description"
+                            compactButtons
+                            maxLength={600} // No limit on backend model, but enforce shortish description
+                            paywall={!hasAvailableFeature(AvailableFeature.INGESTION_TAXONOMY)}
+                            saveButtonText="Set"
+                        />
+                        {hasAvailableFeature(AvailableFeature.TAGGING) && (
+                            <ObjectTags
+                                tags={action.tags ?? []}
+                                onTagSave={(tag) => {
+                                    saveNewTag(tag)
+                                    setEdited(true)
+                                }}
+                                onTagDelete={(tag) => {
+                                    deleteTag(tag)
+                                    setEdited(true)
+                                }}
```

Comment:
Why are these two separate actions? Can't this use the traditional `value`/`onChange` pattern that we have on most controlled components?